tests/samples.ts

```diff
@@ -4,7 +4,33 @@ import {
    ApplicationLoadBalancerEventRequestContext,
    APIGatewayRequestEvent,
    ApplicationLoadBalancerRequestEvent } from '../src/request-response-types';
-import { Context } from 'aws-lambda';
+import { APIGatewayEventIdentity, Context } from 'aws-lambda';
+import withDefault from './test-utils/withDefault';
+
+interface MakeAPIGatewayRequestEventContextParams {
+   httpMethod?: string;
+}
+
+interface MakeAPIGatewayRequestEventHeadersParams {
+    [name: string]: string | undefined;
+}
+
+interface MakeAPIGatewayRequestEventMultiValueHeaderParams {
+   [name: string]: string[] | undefined;
+}
+
+interface MakeAPIGatewayRequestEventMultiValueHeaderParamsInput {
+   headers?: MakeAPIGatewayRequestEventHeadersParams;
+   multiValueHeaders?: MakeAPIGatewayRequestEventMultiValueHeaderParams;
+}
+
+interface MakeAPIGatewayRequestEventParams {
+   httpMethod?: string;
+   path?: string;
+   headers?: MakeAPIGatewayRequestEventHeadersParams;
+   multiValueHeaders?: MakeAPIGatewayRequestEventMultiValueHeaderParams;
```

Comment:
Able to add comments explaining why using `headers` is preferred, but `multiValueHeaders` can be used to set multiple values for specific headers?